docs/CRSF.md

```diff
@@ -2,86 +2,52 @@
 
 ([back to main page](../README.md))
 
-This page describes how to set up a mLRS system for EdgeTX/OpenTX radios, so that you get the usual CRSF telemetry sensors and can use the Yaapu telemetry app.
+This page describes how to set up an mLRS system for EdgeTX/OpenTX radios, so you get CRSF telemetry sensors and can use the Yaapu telemetry app.
 
-Two things need to be done:
-1. The mLRS Tx module needs to be put into "CRSF mode"
-2. The flight controller needs to be set up to send a MAVLink stream with the desired MAVLink messages
+Three steps need to be completed:
+1. The transmitter needs to be setup for the CRSF protocol
+2. The mLRS Tx module needs to be put into "CRSF mode"
+3. The flight controller needs to be set up to send a MAVLink stream with the desired MAVLink messages
 
 Optional but recommended steps:
+- Install the mLRS Lua script on the transmitter to configure Tx and Rx parameters: [Lua Script](LUA.md)
 - Set the receiver into "mavlink mode" (described below)
-- Install the mLRS LUA script on the radio which allows you to change the Tx and Rx parameters: ([LUA Script](LUA.md))
+- Install the Yaapu app (described below)
 
 Notes:
 - Any radio which supports the CRSF protocol should work, this should include many brands besides EdgeTX/OpenTX radios.
 - An ArduPilot flight controller is assumed. PX4 and INAV needs to be tested and validated.
 
+## Transmitter Setup
+
+In EdgeTX/OpenTX, navigate to MDL->MODEL SETUP and configure the external RF module for CRSF protocol with 400K baud rate. 
 
 ## mLRS Tx Module Setup
 
+Set the following parameters using the CLI or Lua script:
+
 - Tx Ch Source = crsf
 - Tx Ser Dest = serial or serial2 (not mbridge!)
 - Tx Ser Link Mode = mavlink
-- Tx Snd RadioStat = off (yes, off!)
-
-Notes: 
-- There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing. You really should not need it for this setup.
-- The choice of Tx Ser Baudrate is uncritical and really determined by the user's need. It is recommended to set it to 57600 or higher, as this will provide enough speed for all operation modes (19 Hz, 31 Hz, 50 Hz).
-
 
 ## ArduPilot Setup
 
-Configuration of a serial port for MAVLink v2:
```

Comment:
here we could have a statment and refernce to ArduPilot page

Reply:
Added